src/akshita/CharacterOperations.java

```diff
@@ -0,0 +1,27 @@
+//WAP to print string "Technocredit" in reverse order with frequency of repeated Char
+
+
+package akshita;
+
+public class CharacterOperations {
+	void reverseWithFrequency(String input,char var,int count) {
+		System.out.println("print string Technocredit in reverse order with Index");
+			for(int index=input.length()-1;index>=0;index--) {
+				System.out.println(index + " => " + input.charAt(index)); 
+					if(input.charAt(index) == var) {
+						count = count +1;
+				}
+		}
+	System.out.println("Frequency of " + var + " is " + count );	
+	}
+	
+public static void main(String[] args) {
+	CharacterOperations characterOperations = new CharacterOperations();
```

Comment:
create 2 separate methods for both programs, here reverse order of technocredits is printing each time, if we want to check frequency of any character.